Ports/quake3/patches/0001-Meta-Refactor-Makefile-to-support-Serenity.patch

```diff
@@ -0,0 +1,133 @@
+From 702293dcfc074c7d0bec034bcc8d477ff1799cf4 Mon Sep 17 00:00:00 2001
+From: Jesse Buhagiar <jooster669@gmail.com>
+Date: Fri, 25 Mar 2022 09:39:21 +1100
+Subject: [PATCH 1/8] Meta: Refactor Makefile to support Serenity
+
+---
+ Makefile | 37 +++++++++++++++++++------------------
+ 1 file changed, 19 insertions(+), 18 deletions(-)
+
+diff --git a/Makefile b/Makefile
+index 9e535559..027bd264 100644
+--- a/Makefile
++++ b/Makefile
+@@ -3,8 +3,8 @@
+ #
+ # GNU Make required
+ #
+-COMPILE_PLATFORM=$(shell uname | sed -e 's/_.*//' | tr '[:upper:]' '[:lower:]' | sed -e 's/\//_/g')
+-COMPILE_ARCH=$(shell uname -m | sed -e 's/i.86/x86/' | sed -e 's/^arm.*/arm/')
++COMPILE_PLATFORM=serenity
++COMPILE_ARCH=amd64
```

Comment:
This means only the x86_64 build will work right? can we have this pull `$SERENITY_ARCH` ?

Reply:
I'm not sure, I'll double check. I think the `makefile` has heavy use of `amd64` and not `x86_64` so I'll take it for a spin.